strr-web/interfaces/application-i.ts

```diff
@@ -44,6 +44,7 @@ export interface ApplicationDetailsI {
   primaryContact: ContactI
   secondaryContact?: ContactI
   principalResidence: PrincipalResidenceI
+  propertyManager: PropertyManagerI
```

Comment:
I think propertyManager should be optional

Reply:
Good catch, yea it should be optional.

Reply:
Updated